config/prod.exs

```diff
@@ -1,9 +1,17 @@
 use Mix.Config
 
-config :logger, level: :error
-
 config :mozart_fetcher, timeout: 3000
 
 config :ex_metrics,
   send_metrics: true,
   raise_on_undefined_metrics: false
+
+config :logger,
+  backends: [{LoggerFileBackend, :file}]
+
+config :logger, :file,
+  path: "/var/log/mozart_fetcher/error.log",
+  format: "$message\n",
```

Comment:
as the format is set in the `config.exs` - I don't think we need to set it here too?

Reply:
Because the other one is setting the log format for the console and this one is setting it for the file backend